extensions/amp-story/1.0/amp-story.js

```diff
@@ -1142,6 +1180,10 @@ export class AmpStory extends AMP.BaseElement {
    * @private
    */
   onPausedStateUpdate_(isPaused) {
+    if (!this.activePage_) {
```

Comment:
when would this case happen? i.e. when would a non-active page be paused?

Reply:
This is actually the case where we pause the story before the pages are initialized. ``this.activePage_`` here refers to the currently displayed page, which might not already be set.